lib/msal-browser/package.json

```diff
@@ -87,7 +87,7 @@
     "ts-jest": "^27.0.2",
     "tslib": "^1.10.0",
     "tslint": "^5.20.0",
-    "typescript": "^3.8.3"
+    "typescript": "^4.9.5"
```

Comment:
This will most certainly contain breaking changes that will affect our build output. Needs to wait until v3

Reply:
And we should be using ~ versions as typescript doesn't follow semver

Reply:
cc @hectormmg @konstantin-msft Let me check the release notes before we make the call if we need to make this a part of v3.